### ISSUE TITLE
Doesn't find all directories

### ISSUE THREAD

**Issue by 677230756E64** [ffuf/ffuf#706]
Enumerating the Kioptrix #1 machine, the default scan of
`ffuf -w <wordlist> -u http://<machine>/FUZZ` 
doesn't return all the directories, some are clearly missing
for some reason adding a slash after FUZZ (not mentioned anywhere and probably not intended) somewhat solves it, but not without weird quirks like displaying
`cgi-bin
cgi-bin/` as two different directories. 
any idea why the tool works like that?

**Comment by bsysop** [comment]
Hi @677230756E64, do you have an external resource that I could use to validate this scenario?

Thanks

**Comment by 677230756E64** [comment]
> Hi @677230756E64, do you have an external resource that I could use to validate this scenario?
> 
> Thanks

Yes, download the Kioptrix 1 machine and scan it. First with something like dirbuster, which I found to be the most thorough, then with ffuf. Let me know of the results and the options you've used for ffuf as well as the wordlists used. I used built-in wordlists on kali, both from dirb and dirbuster directories. For me ffuf never managed to find some directories on that particular machine like icons/, doc/, cgi-bin/

**Comment by karthikuj** [comment]
Is the resolution for this in progress? Adding a forward slash after FUZZ solves it but then -recursion won't work.

**Comment by bsysop** [comment]
Indeed @karthikuj 

Or you can create a new wordlist only for directories, adding a `/`, and then it will probably work as you need.
```bash
cat your_wordlist.txt | sed 's|$|/|g' > new_wordlist.txt
```

I hope that helps, closing the issue by now, feel free to re-open if needed.

Happy hacking.
